### PR TITLE
Update branding

### PR DESCRIPTION
**Pull request by RussKie**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2414)

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -4,7 +4,11 @@
     <!-- version in our package name Ver.sion.Prefix-PreReleaseVersionLabel.#####.## -->
     <!-- where #####. come from the date and .## is the build-per-day -->
     <VersionPrefix>4.8.1</VersionPrefix>
```
Comment (wtgodbe) [inline eng/Versions.props]: Does this also need to be bumped?

**Comment by wtgodbe** [comment]
I believe you also need an internal trigger here: https://github.com/dotnet/winforms/blob/89087f8bf580cacd622ad215aaaa131001b9a10a/azure-pipelines.yml#L13-L16

@mmitche please confirm

File: eng/Versions.props (hunk 1 above)
Comment (RussKie) [reply eng/Versions.props]: I don't believe so.
@mmitche @AdamYoblick

**Comment by RussKie** [comment]
> I believe you also need an internal trigger here:

Done

File: eng/Versions.props (hunk 1 above)
Comment (AdamYoblick) [reply eng/Versions.props]: Honestly, I'm not sure. We haven't bumped this in a bit, since the date and the build number will always increment.

Comment (mmitche) [reply eng/Versions.props]: I think that verison is correct, and 3.0.x is 4.8.0.

**Comment by RussKie** [comment]
@wtgodbe I'm leaving this with you to merge

**Comment by AdamYoblick** [comment]
Looks like the internal build failed https://dev.azure.com/dnceng/internal/_build/results?buildId=440146

@wtgodbe Is this due to needing an arcade update in the 3.1 release branch?

**Comment by wtgodbe** [comment]
I think we just need to update nuget.config to use the new feed for arcade – I put up a PR: https://github.com/dotnet/winforms/pull/2434

**Comment by AdamYoblick** [comment]
Thank you, I approved it :)

**Pull request merged**